operators/cuda/negxplus1.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include "ocos.h"
+#include "negxplus1_impl.cuh"
+
+namespace contrib {
+
+template <typename T>
+struct NegXPlus1 {
+  template <typename TDict>
+  OrtStatusPtr OnModelAttach(const TDict& /*dict*/) {
+    return nullptr;
+  }
+  OrtStatusPtr Compute(Ort::Custom::CUDAKernelContext* ctx,
```

Comment:
If you want to support Eager mode debugging, you need to return OrtxStatus, instead of OrtStatusPtr here. But we can replace all of them later.

Reply:
I'll do that in a different PR.